L/Libtask/build_tarballs.jl

```diff
@@ -1,52 +1,58 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using Pkg
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
-name = "Libtask"
-version = v"0.3.2"
-commit_id = "fbe338053f402d76524d0a01f3796dd7da90b781"
+julia_version = v"1.3.1"
 
-# see https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/336
-# ENV["CI_COMMIT_TAG"] = ENV["TRAVIS_TAG"] = "v" * string(version)
+name = "Libtask"
+version = VersionNumber("0.4.0-julia.$(julia_version.major).$(julia_version.minor)")
```

Comment:
Unfortunately this trick doesn't work, pre-release versions like this are apparently rejected by the registry (see discussion on #1948).

Reply:
Ah I knew there would be some problem, I was just too lazy to read through the whole discussion :smile:

Reply:
So the workaround is basically to manually map different versions of Libtask to different Julia versions?

Reply:
Yeah, I think so. As I understand it, the long-term plan for JLLs is to decouple the JLL version from the upstream software version anyway (as a lot of software has versioning schemes which are difficult to accurately squeeze into semver)